plugins/catalog-backend-module-azure/report.api.md

```diff
@@ -15,6 +15,25 @@ import { SchedulerService } from '@backstage/backend-plugin-api';
 import { SchedulerServiceTaskRunner } from '@backstage/backend-plugin-api';
 import { ScmIntegrationRegistry } from '@backstage/integration';
 
+// @public
+export class AzureBlobStorageEntityProvider implements EntityProvider {
```

Comment:
Let's drop this export. If we only support the new backend system we should be able to stop exporting this class and have a slimmer public API surface that we can evolve more easily

Reply:
so if I dont export this class from index.ts file it is not reflected in api report. Sorry, i am confused like do we want this class in api report or not.